docs/debugging.md

```diff
@@ -85,7 +85,7 @@ dashboard](https://happo.io/dashboard), look under "Snap-requests" and click on
 the one that has a failure icon.
 
 ### `Timed out while waiting for window.happo`
-A common error when using [the Storybook integration](storybook.md) is `Timed
-out while waiting for window.happo`. This is often caused by missing to
+
+A common error when using [the Storybook integration](storybook.md) is `Timed out while waiting for window.happo`. This is often caused by missing to
```

Comment:
There are a couple of these now. Looks like prettier won't linebreak within backticks. I'd expect it to wrap either before or after instead, but 🤷‍♂️